Cargo.toml

```diff
@@ -13,6 +13,7 @@ license = "MIT"
 name = "rusqlite"
 
 [features]
+default = ["backup", "blob", "chrono", "functions", "load_extension", "serde_json", "trace"]
```

Comment:
I don't think I want to enable all features by default just to get docs.rs to show everything. Let's table this until onur/docs.rs#29 is addressed (so we can enable these just for the docs and not for all users).